scripts/detect_timeout.py

```diff
@@ -1,7 +1,45 @@
 import argparse
 import re
+from nltk import word_tokenize
 from pathlib import Path
 
+#path to directory containing error logs as .txt files
+error_p = Path("__file__").resolve().parents[0] / 'adni_error_logs'
+
+log_list = []
+out_memory_list = []
+time_limit_list = []
+trait_list = []
+
+for file in error_p.iterdir(): #for every file in the path
+    log_list.append(file) #save the file's name in the list_files list
+    with open(file) as f:
+        f_lines = f.readlines() #reads over each line of the file
+
+    for line in f_lines: #for each line of the file
+        split_line = line.split() #splits the contents of each line according to white space.
+
+        #loop through tokens and see if they correspond to common error terms. If so, add file name to list
+        for token in split_line:
+            if token == 'out-of-memory':
+                out_memory_list.append(file.stem)
+
+            if token == 'LIMIT': 
+                time_limit_list.append(file.stem)
+
+            if token == 'FunctionalSummaryInputSpec':
+                trait_list.append(file.stem)
```

Comment:
It's great that you have identified a lot of the keywords! 
Let's work on integrating all these into the `check_timeout` function below. We can certainly rename the function to `check_error_logs` or something.

Reply:
You will need to integrate this block to line 62. It can be a bunch of if statements for now.